src/analysis/mod.rs

```diff
@@ -125,6 +129,38 @@ fn analyze_global_functions(env: &mut Env) {
     });
 }
 
+fn analyze_constants(env: &mut Env) {
+    let ns = env.library.namespace(library::MAIN_NAMESPACE);
+
+    let full_name = format!("{}.*", ns.name);
+
+    let obj = match env.config.objects.get(&*full_name) {
+        Some(obj) if obj.status.need_generate() => obj,
+        _ => return,
+    };
+
+    let constants: Vec<_> = ns.constants
+        .iter()
+        .collect();
+    if constants.is_empty() {
```

Comment:
Strange why this check not works for gdk

Reply:
Seems need early return after `constants::analyze` too

Reply:
I'll take a look :)